### PR TITLE
Adding more info on how to set up the models

### PR DESCRIPTION
**Pull request by Mihai-P**
Should we move the Usage before what I have added? Also should the root in my example be integer or should be the same type as the primary key of the model?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +69,57 @@ class TreeQuery extends \yii\db\ActiveQuery
...
+    ```
+
+2. `rgt`, type integer. To change the name of the field make the same setting as for lft,
+but use the attribute `rightAttribute`
+
+3. `depth`, type integer. To change the name of the field make the same setting as for lft,
+but use the attribute `depthAttribute`
+
+
+These fields should be set up in the rules section of the model like this.
```
Comment (creocoder) [inline README.md]: @Mihai-P Fields should not be listed inside `rules()`.

File: README.md [hunk 2]
```diff
@@ -68,6 +69,57 @@ class TreeQuery extends \yii\db\ActiveQuery
...
+    return [
+        'nestedSets' => [
+            'class' => NestedSetsBehavior::className(),
+            'treeAttribute' => 'root',
+        ],
+    ];
+}
+```
+
+Add this field to the rules function in the same way as the ones for the single tree.
```
Comment (creocoder) [inline README.md]: @Mihai-P Same here.

**Comment by Mihai-P** [comment]
I will change the order of the information like the one in the laravel package, it makes a lot more sense. I know it is not the usual order but right now it is a little all over the place.

**Comment by Mihai-P** [comment]
I changed the order of the documentation to match the one in the Laravel package, it makes more sense then having Installation then Usage then Configuration.

**Comment by creocoder** [comment]
@Mihai-P Thanks! Great work!

**Pull request merged**